app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,8 @@
+class DishIngredientsController < ApplicationController
```

Comment:
Nice controller organization! Great work setting up this controller to handle operations on this joins resource